### DISCUSSION TITLE
Deleting Bonds

### DISCUSSION THREAD

**Discussion by pwesthoffirhythm** [capacitor-community/bluetooth-le#480] [General]
Hi! I was wondering if there was a way to delete bonds that I'm missing or if that was something that could be added?

I assume this is only possible for android as I doubt that functionality is exposed by iOS.  I know that it's possible in android as the NRF connect app is able to delete bonded devices and I see that the device is subsequently removed from the list of saved devices in the android bluetooth tile.  This option is only offered if the device was paired using the NRF connect app.

**Comment by pwespi** [comment]
Hi, thank you for using this plugin.

In Android it is in theory possible to remove a bond, but the API is hidden and not guaranteed to be stable. I think the intended way is to remove bonds from the system settings.

You are right, on iOS it's not possible.
